pages/common/go-mod.md

```diff
@@ -0,0 +1,16 @@
+# go mod
+
+> Tool for module maintenance.
+> More information: <https://golang.org/cmd/go/>.
+
+- Download modules to local cache:
```

Comment:
Let's start with "go mod init", then "go mod tidy", "go mod download", "go mod vendor", and then "go mod verify".